client/index.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import $ from 'jquery';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      images: []
+    };
+  }
+
+  componentDidMount() {
+    $.ajax({
+      method: 'GET',
+      url: 'http://localhost:3000/images',
```

Comment:
better to use a relative path instead of the absolute path otherwise it won't work when deploying the web app later.

Reply:
Great! Thank you for your advices!